src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const mainPrice = 40;
+  let totalPrice = days * mainPrice;
+  const smallestDiscount = 20;
+  const biggestDiscount = 50;
+
+  if (days >= 7) {
+    totalPrice -= biggestDiscount;
+  }
+
+  if (days >= 3 && days < 7) {
+    totalPrice -= smallestDiscount;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    totalPrice -= biggestDiscount;
  } else if (days >= 3) {
    totalPrice -= smallestDiscount;
  }
```
You can use "else if" to make code shorter.
```suggestion
  if (days >= 7) {
    return totalPrice - biggestDiscount;
  }

  if (days >= 3 ) {
    return totalPrice - smallestDiscount;
  }
```

or like this

Reply:
I like second variant)